### PR TITLE
Add memorial banner for Queen

### PR DESCRIPTION
**Pull request by ebrett**
## Ticket and context

Ticket: [ER-409](https://dfedigital.atlassian.net/browse/ER-409)

### CODE REVIEW

File: app/views/layouts/landing_page_layout.html.erb [hunk 1]
```diff
@@ -55,6 +55,23 @@
             Help for early years providers
           </a>
         </div>
+        <div>
```
Comment (agoldstone93) [inline app/views/layouts/landing_page_layout.html.erb:58]: Is this empty `div` needed?

File: app/views/layouts/landing_page_layout.html.erb [hunk 2]
```diff
@@ -55,6 +55,23 @@
...
+          <div class="gem-c-emergency-banner" aria-label="emergency banner" role= "region" data-nosnippet="true">
+            <div class="govuk-width-container">
+              <div class="govuk-grid-row">
+                <div class="govuk-grid-column-two-thirds">
+                  <h2 class="gem-c-emergency-banner__heading">Her Majesty Queen Elizabeth II</h2>
+                  <p class="gem-c-emergency-banner__description">
+                    21 April 1926 to 8 September 2022
+                  </p>
+                  <a href="" class="gem-c-emergency-banner__link">
```
Comment (agoldstone93) [inline app/views/layouts/landing_page_layout.html.erb:67]: `href` is empty. Should it have a link in it?

**Review by agoldstone93 — APPROVED** [review]
Small changes / questions

File: app/views/layouts/landing_page_layout.html.erb (hunk 2 above)
Comment (ebrett) [reply app/views/layouts/landing_page_layout.html.erb:67]: woops, yes it should

**Pull request merged**